frontend/src/js/constants/Constants.js

```diff
@@ -107,15 +108,149 @@ const JOYRIDE_STEPS = [
             "Use this as an overlay or as an in-cell side-by-side comparison.",
             '</br><span style="color: red;">Clear Cell</span> will reset the cell back to the default. ',
             "This can be undone if you accidentally click it with the undo button.",
-            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.'
+            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.',
+            '</br><span style="color: purple;">Export</span> allows you to export/save the plot. NOT Implemented.',
+            '</br><span style="color: orange;">Calculator</span> allows you to create a new variable based on',
+            ' calculations performed on another variable.'
         ),
         selector: ".tools-container",
         position: "right",
         type: "click"
+    },
+    {
+        title: "Calculator",
+        text: "This button opens the calculator tool! Cool right?! Yeah it's cool :p",
+        selector: "#open-calculator-button",
+        position: "right",
+        type: "hover"
     }
 ];
 
+const CALCULATOR_STEPS = [
+    {
+        title: 'Calculator Tour',
+        text: 'This is a tour of the calculator and how to use it!',
+        selector: '#calculator-main',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Variables List',
+        text: "All the available variables that you've loaded are listed here.",
```

Comment:
"All the available variables are listed here."
or
"All the available variables that you have loaded are listed here."